packages/react-components/src/components/ChatMessage/ChatMessageContent.tsx

```diff
@@ -174,16 +174,12 @@ const messageContentAriaText = (props: ChatMessageContentProps): string | undefi
     : undefined;
 };
 
-type NodeProcessInstruction = {
-  shouldProcessNode: unknown;
-  processNode: unknown;
-};
+const processNodeDefinitions = ProcessNodeDefinitions();
+const htmlToReactParser = Parser();
 
 const processHtmlToReact = (props: ChatMessageContentProps): JSX.Element => {
-  const htmlToReactParser = new Parser();
-
   /* @conditional-compile-remove(teams-inline-images) */
-  const processInlineImage: NodeProcessInstruction = {
+  const processInlineImage: ProcessingInstructionType = {
```

Comment:
This processInlineImage instance will be created as a new instance everytime when you call processHtmlToReact, consider extract it out

Reply:
Would be nice to get these 2 process objects out of the function, it makes this function too long to read, and I am expecting more process instruction will be coming in the future